pkg/log/fluentd.go

```diff
@@ -0,0 +1,25 @@
+package log
+
+import (
+	"github.com/evalphobia/logrus_fluent"
+	"github.com/sirupsen/logrus"
+)
+
+// NewFluentHook return a fluentd hook for logrus loggin
+func NewFluentHook(level logrus.Level) (*logrus_fluent.FluentHook, error) {
+	// add fluentd hooks if specified in configuration
```

Comment:
Either I do not see Traefik in terms of (I am familiar with whole traefik codebase), or the comment "if specified in configuration" seems to me also (yet) incorrect.

Reply:
This is WIP, and it was a comment left here as a reminder. For now, everything is done in https://github.com/bearstech/traefik/tree/add-fluentd-draft in order to discuss and find the best implementation for this feature.

Reply:
@papey to avoid this kind of issue, could close the PR for now?

Reply:
Sure.

Reply:
thank you